tests/testthat/test-util.R

```diff
@@ -173,6 +173,19 @@ test_that("git", {
                info[c("sha_short", "sha", "branch")])
 })
 
+test_that("git_info non-cran", {
+  path <- unzip_git_demo()
```

Comment:
sorry, looks like this needs a `skip_on_cran()` as the `unzip_git_demo` will otherwise not work. Should also add a `skip_if_no_git` in here too